src/main/java/org/mdbenefits/app/utils/SubmissionUtilities.java

```diff
@@ -128,6 +128,24 @@ public static List<String> getHouseholdMemberNames(Submission submission) {
         return names;
     }
 
+    public static String householdMemberFullNameFormatted(Map<String, Object> householdMember) {
+        String fullName = String.format("%s, %s", householdMember.get("householdMemberLastName"), householdMember.get("householdMemberFirstName"));
+        if (householdMember.get("householdMemberMiddleName") != null) {
+            fullName += ", " + householdMember.get("householdMemberMiddleName");
+        }
+        return fullName;
+    }
+
+    public static String applicantFullNameFormatted(Submission submission){
+        Map<String, Object> inputData = submission.getInputData();
+
+        String fullName = String.format("%s, %s", inputData.get("lastName"), inputData.get("firstName"));
+        if (inputData.get("middleName") != null) {
+            fullName += ", " + inputData.get("middleName");
```

Comment:
I've never seen a comma between the first and middle name when using this format. It's usually `Jones, James Earl`. That would mean the code should be:

```suggestion
            fullName += " " + inputData.get("middleName");
```